parseAPI/h/CFG.h

```diff
@@ -341,7 +341,7 @@ class PARSER_EXPORT Block :
     void getFuncs(std::vector<Function *> & funcs);
     template<class OutputIterator> void getFuncs(OutputIterator result); 
 
-    void getInsns(Insns &insns) const;
+    virtual getInsns(Insns &insns) const;
```

Comment:
Compile error: the return type is missing.

Reply:
Fixed. Thanks!